README.md

```diff
@@ -65,6 +65,18 @@ cipher encrypt/decrypt AES CTR multipart:
         success!
 ```
 
+Any subsequent run where entropy is already present, or if entropy injection is
+not enabled, will result in the following acceptable output:
+```
+warning (-133) - this attempt at entropy injection failed
```

Comment:
It is not clear to me what is the acceptable output.
Should `warning (-133) - this attempt at entropy injection failed` appear before any of the next success lines? Only before AES CBC?

Reply:
The warning will always be printed before other things.